### DISCUSSION TITLE
brew install --cask: search for similarly named casks

### DISCUSSION THREAD

**Discussion by jrmann100** [Homebrew/discussions#3986] [Casks] [answered]
When I mistype a formula using `brew install`, Homebrew searches for "similarly named formulae" and recommends the closest one after getting a "Warning: No available formula" error.

Is it possible to enable a similar functionality when using `brew install --cask`? Usually, if I am nearly confident in a cask name, I will run a search while typing out the install command while it is running. But it seems redundant to run both commands each time.

This appears similar to an old issue/PR in 2018: https://github.com/Homebrew/brew/issues/3829

**Comment by SMillerDev** [comment]
Looking at that issue this should work already

**Comment by jrmann100** [comment]
This does not appear to be the case, else I am using/understanding Homebrew wrong.
<img width="610" alt="image" src="https://user-images.githubusercontent.com/17690438/204101416-9ab30044-6566-4552-9171-58a2e9264678.png">
As seen in the image: using `--cask` I get no suggestions; without `--cask` I only get suggestions for formulas (as expected).
<details>
  <summary>My brew config</summary>

```
HOMEBREW_VERSION: 3.6.12-20-g9c88c39
ORIGIN: https://github.com/Homebrew/brew
HEAD: 9c88c39baead6bbc40b12ab1eae9b63bcb5a8042
Last commit: 2 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 794ddefeeadaddb3977434d23a9c2597b23f8652
Core tap last commit: 6 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.38.1 => /opt/homebrew/bin/git
Curl: 7.84.0 => /usr/bin/curl
macOS: 13.0.1-arm64
CLT: 14.1.0.0.1.1666437224
Xcode: 14.1
Rosetta 2: false
```

</details>

**Reply by SMillerDev** [reply]
That actually sounds like a bug, could you report it to Homebrew/brew?

**Reply by jrmann100** [reply]
@SMillerDev, just to confirm: you'd like me to file a bug report, not a feature request, at [Homebrew/brew](https://github.com/Homebrew/brew), and not [Homebrew/homebrew-cask](https://github.com/homebrew/Homebrew-cask)?

**Reply by SMillerDev** [reply]
Yes, Homebrew/cask only stores the package definitions so there's not much it can do. Homebrew/brew is in control of `brew` behaviour.

**Comment by jrmann100** [comment, marked as answer]
Opened https://github.com/Homebrew/brew/issues/14183.
